src/routes/Apps/App/Testing.tsx

```diff
@@ -535,6 +550,12 @@ class Testing extends React.Component<Props, ComponentState> {
                         </OF.PivotItem>
                     </OF.Pivot>
                 </div>
+                <ConfirmCancelModal
+                    open={this.state.isConfirmClearModalOpen}
```

Comment:
What is the reason for the name mismatch?
The modal is `ConfirmCancelModal` but the boolean is `isConfirmClearModalOpen`.
Is there difference between semantics of Cancel vs Clear here?
It would expect `isConfirmCancelModalOpen`

Reply:
The component name is "ConfirmCancelModel" (used in a lot of places).   The confirmation itself is a warning when the user want to clear the results (i.e. a specific instance of this modal)